uaclient/entitlements/esm.py

```diff
@@ -95,10 +95,10 @@ class ESMInfraEntitlement(ESMBaseEntitlement):
 
     @property
     def repo_pin_priority(self) -> Optional[str]:
-        """Once a release goes into EOL it is entitled to ESM Infra."""
-        if system.is_active_esm(system.get_platform_info()["series"]):
+        """All LTS are entitled to ESM Infra."""
+        if system.is_lts(system.get_platform_info()["series"]):
```

Comment:
Yes an I think this represents a product change in behavior that happened a couple years ago that wasn't accurately reflected in the APT preferences setup for esm-infra